app/components/event-card.js

```diff
@@ -6,6 +6,18 @@ import { pascalCase } from 'open-event-frontend/utils/string';
 export default Component.extend({
   classNames: ['column'],
 
+  categoryTag: computed('event.topic', function() {
```

Comment:
Why does this have to be a computed property ? You could directly use `event.topic` right ?

Reply:
Actually I tried this earlier but I didn't get the filtered events on clicking. I explored other files and found out this method which worked perfectly fine. 
Ref: `app/components/forms/orders/attendee-list.js`

Reply:
This is incorrect. You're basically having a computed property as an alias to another property. 

> Actually I tried this earlier but I didn't get the filtered events on clicking

Then there could be something else causing it. This is not a valid fix.

Reply:
Ok. I'll make the suggested changes

Reply:
@niranjan94 I tried what you suggested but it didn't work. Please help

Reply:
@niranjan94 @CosmicCoder96 please help